content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,167 @@
+- Flame graph: visualize aggregated profile data
+- Find resource consumption broken down by method name and line number
+- Insights available from various staging and production environments
+
+## Requirements
+
+- **A Datadog account**: The plugins require a datadog account. If you're new to the Datadog suite of products, go to the [Datadog website][4] to learn more about Datadog's observability tools and sign up for a free trial.
+- **Continuous Profiling**: To display code-level insights, the plugin requires Continuous Profiling instrumented on your Java Services. For more information, see [Getting Started with the Continuous Profiler][3].
+
+## Setup
+
+### Install the Datadog plugin
+
+1. Click **Plugins** and search for `Datadog`.
+1. Click **Install** to download and install the plugin in your IDE.
+1. If you receive a prompt notifying you that Datadog is a third-party plugin, click **Accept**.
+1. Click **Restart IDE** to restart IDEA.
+
+{{< img src="/developers/ide_integrations/idea/datadog-plugin1.png" alt="The Datadog plugin" style="width:100%;" >}}
+
+### Add Datadog credentials
+
+After installing the Datadog plugin and restarting IDEA, add your Datadog API and application keys to authenticate with Datadog:
+1. With a file or project open in IDEA, click the **Datadog** tool window.
+1. Click **Add your credentials...**.
+1. Enter your name, [API key and application key][5], and select a site.
+
+**Note**: For most users, one pair of keys is all that is required. If you're using a multi-org setup with multiple key pairs, check to ensure that the correct pair is active using the **Tools** -> **Datadog** section in **Preferences**.
+
+### Link a service
+
+To provide relevant data from the Datadog platform, add relevant services to a project:
+1. With your project open in IDEA, open the **Datadog** tool window and click the plus icon (**+**).
+1. Search for and select the services that you want to add to the current project.
+
+To remove a service, select it in the **Services** table and click the minus icon (**-**).
+
+<div class="alert alert-info">The names of linked services persist with the project when you close it.</div>
+
+## Using the plugin
+
+After you add a service to your project, right-click on the service and click **Open in Profiling** to open a Profiling tab for the service. A profiling tab displays data for only one service, but you can have multiple tabs open simultaneously.
+
+The Profiling tab shows Continuous Profiling information for the service in a selected environment, aggregated over a specific time frame. Available views are:
+- [Top list](#top-list): Displays a list of the most resource intensive methods for the current profiling measure.
+- [Flame graph](#flame-graph): A flame graph representing stack traces in the profiles.
+
+{{< img src="/developers/ide_integrations/idea/profiling-example.png" alt="An example of the Profiling tab for a service named product-recommendation" style="width:100%;" >}}
```

Comment:
This image shows text `Flamegraph` and includes the code sync button that will be removed.  A more recent image is https://user-images.githubusercontent.com/1835893/195404809-53b81808-8252-4306-81eb-1274933ef8c5.png